ReactCommon/react/renderer/core/RawPropsKey.cpp

```diff
@@ -23,19 +23,19 @@ void RawPropsKey::render(char *buffer, RawPropsPropNameLength *length)
   if (prefix) {
     auto prefixLength = std::strlen(prefix);
     std::memcpy(buffer, prefix, prefixLength);
-    *length = prefixLength;
+    *length = static_cast<RawPropsPropNameLength>(prefixLength);
   }
 
   // Name
   auto nameLength = std::strlen(name);
   std::memcpy(buffer + *length, name, nameLength);
-  *length += nameLength;
+  *length += static_cast<RawPropsPropNameLength>(nameLength);
```

Comment:
are these right? we are copying nameLength bytes but then advancing length a potentially different number of bytes?
shouldn't the cast happen in the assignment of nameLength instead? same w/others